BaseStation/VideoStreamer/VideoStream.xaml

```diff
@@ -1,9 +1,8 @@
-﻿<Window x:Class="VideoStreamer.MainWindow"
```

Comment:
Making VideoStreamer a window is very inflexible. It can still be placed in its own window, but it should _have_ to be in one.

Reply:
Ill look into how to merge this with the Videostreamer branch

Reply:
Don't worry about that.